Source/IntegrationTests/TestFiles/LitTests/LitTest/logger/CSVLogger.dfy

```diff
@@ -5,9 +5,9 @@
 // The CHECK-NOT directives are a regression test: previously the BoogieXmlConvertor
 // wasn't populating the resource count test result property, and the CSV logger was
 // defaulting to 0 for that column.
-// CHECK-NOT: ExampleWithSplits.*,Passed,.*,0
+// CHECK-NOT: ExampleWithSplits.*,Passed,.*,0,0
```

Comment:
I'd make the new character `*` instead of `0` because this test isn't specifically testing the seed. I don't think it'll matter in practice, but it seems like a better expression of what the test is intended to do.